docs/tools/vdb_table/data/rockset.json

```diff
@@ -109,44 +109,44 @@
   },
   "pricing": {
     "value": "",
-    "source_url": "",
+    "source_url": "https://rockset.com/pricing/",
     "comment": ""
   },
   "in_process": {
-    "support": "",
+    "support": "None",
     "source_url": "",
     "comment": ""
   },
   "multi_tenancy": {
-    "support": "",
+    "support": "full",
     "source_url": "",
     "comment": ""
   },
   "disk_index": {
-    "support": "",
+    "support": "full",
     "source_url": "",
     "comment": ""
   },
   "ephemeral": {
-    "support": "",
+    "support": "None",
     "source_url": "",
     "comment": ""
   },
   "sharding": {
-    "support": "",
+    "support": "full",
     "source_url": "",
     "comment": ""
   },
   "doc_size": {
-    "bytes": 0,
+    "bytes": 41,943,040,
```

Comment:
delete commas, they break the JSON syntax

Reply:
removed